connect/models/asset.py

```diff
@@ -15,7 +15,11 @@
 
 
 class Asset(BaseModel):
-    pass
+    def get_param_by_id(self, identity):
+        return self._get_by_id(self.params, identity)
+
+    def get_item_by_id(self, identity):
```

Comment:
This is deprecated......don't add it or leave there for providers. Please create get_item_by_mpn. That's how vendors works, ids are autogenerated since v13 and in phpruntime this is due backwards compatibility only, if you check documentation all goes by mpn

Reply:
I'll make the PR update soon with get_item_by_mpn replacing get_item_by_id. I understand that get_param_by_id is not deprecated?

Reply:
FYI: get_item_by_id has no business sense for an ISV, they don't know the ids since they don't introduce them in connect, they introduce MPNs and this is what's known by vendor.

Reply:
Makes sense. Thank you!